azurerm/internal/services/network/application_gateway_resource.go

```diff
@@ -988,6 +988,29 @@ func resourceApplicationGateway() *schema.Resource {
 											},
 										},
 									},
+
+									"url_configuration": {
+										Type:     schema.TypeList,
+										Optional: true,
+										MaxItems: 1,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"path": {
+													Type:     schema.TypeString,
+													Optional: true,
+												},
+												"query_string": {
+													Type:     schema.TypeString,
+													Optional: true,
+												},
```

Comment:
could we validate this? or are empty strings valid?

Reply:
Interestingly, empty strings are allowed; I've just tried it in the portal and it's accepted.

![2021-03-18_18-03-23_97d8af6d-fee4-4226-9e97-7ffbd8966a52](https://user-images.githubusercontent.com/563596/111674979-865e9100-8814-11eb-97d5-25060fcd3322.png)

Reply:
👍